cmd/subctl/diagnose.go

```diff
@@ -41,6 +41,11 @@ var (
 
 	diagnoseRestConfigProducer = restconfig.NewProducer()
 
+	diagnoseFirewallTunnelRestConfigProducer = restconfig.NewProducer().
+							WithDefaultNamespace(constants.OperatorNamespace).WithPrefixedContext("remote")
+	diagnoseFirewallNatDiscoveryRestConfigProducer = restconfig.NewProducer().
+							WithDefaultNamespace(constants.OperatorNamespace).WithPrefixedContext("remote")
```

Comment:
It would, but it doesn’t work — a given variable can only be used with a single command’s flags. There’s no shared parent command that could be used with persistent flags either.